app/src/test/java/dev/andrylat/raqimbek/bankingutils/BankingUtilsAppTest.java

```diff
@@ -0,0 +1,52 @@
+package dev.andrylat.raqimbek.bankingutils;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.junit.jupiter.api.Test;
+
+public class BankingUtilsAppTest {
+	
+	CardValidator creditCardHandler = new CardValidator();
+
+    @Test
+    public void shouldReturnStringContainingTwoErrors() {
+        var expected = """
+Card number is invalid.
```

Comment:
It is wrong formatting. It is better to use concatenation of string in such kind of cases.